Source/Parallelization/WarpXComm.cpp

```diff
@@ -13,13 +13,14 @@
 #   include "BoundaryConditions/PML_RZ.H"
 #endif
 #include "Filter/BilinearFilter.H"
-#include "Utils/CoarsenMR.H"
+#include "Utils/Parser/IntervalsParser.H"
```

Comment:
Was this added on purpose?

Reply:
I think it was missing, but could have been prior to rebase. Will remove, too - thx!